src/core/pipelines/manifest/parse_manifest_pipeline_options.ts

```diff
@@ -27,15 +27,14 @@ const { DEFAULT_MAX_MANIFEST_REQUEST_RETRY,
  * @returns {Object}
```

Comment:
This file should not exist anymore ? Isn't it ?

Reply:
Good catch2

Reply:
Done